leetcode/src/121.c

```diff
@@ -0,0 +1,27 @@
+int maxProfit(int* prices, int pricesSize){
+	/* If there is only only one day, profit cannot be made
+	 */
+	if(pricesSize <= 1) {
+        return 0;
+    }
+    int min_element = prices[0];
+    int max_difference = prices[1] - min_element;
```

Comment:
You should initialize `max_difference = 0` so that returning value you do not need to check. I mean:
`int max_difference = 0`. And then replace: `return (max_difference < 0)? 0 : max_difference;` by 
`return max_difference`.